README.md

```diff
@@ -1,4 +1,116 @@
+<h1>MVP for board game on Django</h1>
+
+<h2>Summary</h2>
+Multiplayer game, where each player takes turns drawing a card from a pile, which asks if that player was X what X would they be and lists a number of Xs to vote on, players that voted on the selection with the most votes get a point.
+
+The idea is to build a prototype that doesn’t need to handle the operation in multiplayer to begin with, but can at least handle a random card being drawn, voting on a list and tallying those votes, attributing them to the different players.
+
+<h2>How to use this file</h2>
+
+
+
+
+<h2>Design</h2>
+detailed below are all the steps to the creation of the game in Django
+
+**Step 1** Create a model that holds all the players, that can be interacted with inside the page itself, to add or subtract players.
+
+Attributes of this model:
+- Name: str
+- Score: [int]
```

Comment:
What is meant by this syntax? `[int]`

Reply:
will standarized the rest of the contents on each variable from 'int' to '[int]'